src/sidebar/helpers/groups.js

```diff
@@ -4,7 +4,6 @@
  * @typedef {import('../../types/api').GroupIdentifier} GroupIdentifier
  */
 
-// @ts-expect-error - Ignore error about default-importing a CommonJS module.
 import escapeStringRegexp from 'escape-string-regexp';
```

Comment:
This error no longer occurs due to the addition of `allowSyntheticDefaultImports` to the TS config.